python/cuml/test/experimental/test_explainer_kernel_shap.py

```diff
@@ -82,7 +82,10 @@ def experimental_test_and_log(cu_shap_values,
 @pytest.mark.parametrize("model", [cuml.LinearRegression,
                                    cuml.KNeighborsRegressor,
                                    cuml.SVR])
-def test_exact_regression_datasets(exact_tests_dataset, model):
+@pytest.mark.parametrize("idx", np.arange(100))
```

Comment:
Would be good to a couple tests with real datasets too, since some of the synthetic ones can be a little too straightforward.